src/components/note-list.tsx

```diff
@@ -135,26 +142,11 @@ export function NoteList({ baseQuery = "" }: NoteListProps) {
                   disableTooltip
                   aria-label="Change view"
                   className="h-11 w-11 rounded-md bg-bg-secondary hover:bg-bg-tertiary coarse:h-12 coarse:w-12"
+                  onClick={() => toggleView()}
```

Comment:
[Minor] For a function this simple that's also not used anywhere else, I usually prefer to inline it:

```jsx
onClick={() => {
  if (viewType === 'cards') {
    setViewType('list')
  else {
    setViewType('cards')
  }
}}